src/cloud/project/project-patch.md

```diff
@@ -35,37 +32,165 @@ All patch file names must end with the `.patch` extension.
 
 {% include cloud/note-upgrade.md %}
 
-To apply the latest patches, [install the latest version of {{site.data.var.ct}}]({{site.baseurl}}/cloud/project/ece-tools-update.html).
+The Magento Quality Patches package is a dependency for the {{ site.data.var.mcp-prod }} and {{site.data.var.ct}} packages. To apply the latest patches, you must have [the latest version of {{ site.data.var.ct }}]({{site.baseurl}}/cloud/project/ece-tools-update.html) installed.
```

Comment:
Just checked:  On Cloud, you don't have to install MQP. Both patches and mqp packages will be installed and updated by ece-tools.

Reply:
Removed install instructions in 08ecccf3f30e8d3ea53186c61a239c067d0b0c9b